Source/WebCore/platform/graphics/skia/FontSkia.cpp

```diff
@@ -197,4 +198,63 @@ bool Font::platformSupportsCodePoint(char32_t character, std::optional<char32_t>
     return m_platformData.skFont().getTypeface()->unicharToGlyph(character);
 }
 
+sk_sp<SkTextBlob> Font::buildTextBlob(std::span<const GlyphBufferGlyph> glyphs, std::span<const GlyphBufferAdvance> advances, FontSmoothingMode smoothingMode, bool& enableAntialias, bool& isVertical) const
```

Comment:
I don't like that enableAntialias and isVertical are passed in as references, and only conditionally set. If m_platformData.size() or glyphs.size() is empty, they are not set at all.

I'd rather introduce a specific `computeEnableAntialias()` method and `computeIsVerticalText()` and use them at the calcite to initialize m_enableAntialias, m_isVertical, and then pass those bools here by value, not by reference.